src/main/java/com/google/ehub/servlets/ProfileServlet.java

```diff
@@ -67,30 +70,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     } else {
       String userEmail = userService.getCurrentUser().getEmail();
       UserProfile userProfile = profileData.getUserProfile(userEmail);
-      
+
       if (userProfile == null) {
         JsonObject profileJson = new JsonObject();
         profileJson.addProperty(NEEDS_PROFILE, true);
         response.setContentType("application/json");
         response.getWriter().println(profileJson);
-      } 
-      else {
+      } else {
         response.setContentType("application/json");
-        response.getWriter().println(convertToJson(userProfile));
+        response.getWriter().println(new Gson().toJson(
+            new UserData(userProfile, recommendationUtils.getRecommendedEmails(userEmail))));
```

Comment:
If you were to print this out, what would it look like?

Reply:
It is a very similar JSON to the one before, the new structure is like this:
```
{
  profile: {
   username: "",
   bio: ""
   name: ""
  }
  recommendedUsers: [ //list holding user strings
  ]
}
```